samples/SessionSample/Startup.cs

```diff
@@ -13,36 +13,33 @@ namespace SessionSample
 {
     public class Startup
     {
-        // Uncomment the following line to use the Microsoft SQL Server implementation of IDistributedCache.
-        //public Startup(IHostingEnvironment env)
-        //{
-        //    var builder = new ConfigurationBuilder()
-        //        .SetBasePath(env.ContentRootPath)
-        //        .AddJsonFile("appsettings.json");
+        public Startup(IHostingEnvironment env)
+        {
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(env.ContentRootPath)
+                .AddJsonFile("appsettings.json");
 
-        //    if (env.IsDevelopment())
-        //    {
-        //        builder.AddUserSecrets<Startup>();
-        //    }
+            if (env.IsDevelopment())
+            {
+                builder.AddUserSecrets<Startup>();
+            }
 
-        //    Configuration = builder.Build();
-        //}
+            Configuration = builder.Build();
+        }
 
-        //public IConfigurationRoot Configuration { get; set; }
+        public IConfigurationRoot Configuration { get; set; }
 
         public void ConfigureServices(IServiceCollection services)
         {
-            // Adds a default in-memory implementation of IDistributedCache
-            services.AddDistributedMemoryCache();
+            // Uncomment the following line to use the in-memory implementation of IDistributedCache
+            //services.AddDistributedMemoryCache();
 
-            // Uncomment the following line to use the Microsoft SQL Server implementation of IDistributedCache.
-            // Note that this would require setting up the session state database.
-            //services.AddSqlServerCache(o =>
-            //{
-            //    o.ConnectionString = Configuration["AppSettings:ConnectionString"];
-            //    o.SchemaName = "dbo";
-            //    o.TableName = "Sessions";
-            //});
+            services.AddSqlServerCache(o =>
```

Comment:
Sorry, I meant for this section to stay commented out, this is the only part that changes app behavior.

Reply:
Yep, but this need the constructor to be commented out too

Reply:
Why? All the constructor does is create a config object, that's harmless.